### PR TITLE
update README.md and add 'cursoredFind' info + change some format and…

### PR DESCRIPTION
**Pull request by cyclops24**
Hi guys,
It's my first PR (small fix in README.md).
This PR is related to this issue: https://github.com/inProgress-team/react-native-meteor/issues/172

**/CC: ** @noris666 
**/CC: ** @Mokto

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,21 +188,32 @@ class Orders extends Component {
 
 These variables can be used inside getMeteorData or createContainer. They will be populated into your component if they change.
 
-* [Meteor.subscribe](http://docs.meteor.com/#/full/meteor_subscribe) : returns an handle. !! If the component which called subscribe is unmounted, the subscription is automatically canceled.
-* Meteor.collection(collectionName)
-  * [.find(selector, options)](http://docs.meteor.com/#/full/find)
```
Comment (simonbelanger) [inline README.md]: @cyclops24 i think the indentation was intentional

File: README.md [hunk 2]
```diff
@@ -186,21 +188,32 @@ class Orders extends Component {
...
+[Meteor.subscribe()](http://docs.meteor.com/#/full/meteor_subscribe) returns an handle. If the component which called subscribe is unmounted, the subscription is automatically canceled.
+## Meteor.collection(collectionName, options)
+You need pass the `cursoredFind` option when you get your collection if you want to use cursor-like method:
+
+```‍‍‍javascript
+Meteor.collection("collectionName", { cursoredFind: true })
+```
+
+Or you can simply use `find()` to get an array of documents. The option default to false for backward compatibility. Cursor methods are available to share code more easily between a react-native app and a standard Meteor app.
+
```
Comment (simonbelanger) [inline README.md]: I would move these subsections (`Meteor.subscribe` and `Meteor.collection(collectionName, options)`) to the API section

File: README.md (hunk 1 above)
Comment (cyclops24) [inline README.md]: @simonbelanger which indentation you mention?

File: README.md (hunk 2 above)
Comment (cyclops24) [inline README.md]: @simonbelanger Done man

File: README.md (hunk 1 above)
Comment (simonbelanger) [inline README.md]: 2016-12-20 at 15 15
-------
2016-12-20 at 15 17

Comment (cyclops24) [inline README.md]: OK @simonbelanger I also fix that man. 😉

**Pull request merged**